### DISCUSSION TITLE
Post your ideas for "main menu tips" here!

### DISCUSSION THREAD

**Discussion by peppy** [ppy/osu#26212] [Ideas]
We show a random tip on the menu each time the user visits it. Post any suggestions you have here.

You can see existing tips [here](https://github.com/ppy/osu/blob/master/osu.Game/Localisation/MenuTipStrings.cs)!

**Comment by Flamiii** [comment]
Existing tip:

> "Seeking in replays is available by dragging on the difficulty bar at the bottom of the screen!"

This could be changed to: 

> "Seeking in replays is available by dragging on the difficulty bar at the bottom of the screen or by using the left and right arrow keys!"

New tip ideas:

> "You can pause during a replay by pressing Space!"

> "Many of the hotkeys in the game are configurable and can be changed to anything you want!" 

> "If your gameplay HUD is disabled, you can press and hold Ctrl to view it temporarily!"

> "Your gameplay HUD can be customized by using the skin layout editor!"

> "You can create mod presets to make accessing your favorite mod combinations easier!"

> "Many mods now have customization settings that drastically change how they function!"

**Reply by peppy** [reply]
Added;

**Comment by plaguemask** [comment]
"You can press Ctrl-Shift-S to enter the new interactive skin editor!"
"You can press Ctrl-Shift-R to immediately switch to a random installed skin!"

**Reply by peppy** [reply]
Added

**Comment by NotGumballer91** [comment]
> Most mods are customisable. Click on Mod Customisation in mod select to change how they work!

Mods customisibility doesn't come to mind besides Difficulty adjust
> You can press Ctrl-Shift-F anywhere in the game to toggle FPS Counter!

This can probably replace the Ctrl-F11?
> During replay, you can press Ctrl-H to toggle Replay settings!

For those who wants to see a "cleaner" play
> You can easily copy the mods from the score they used by right-clicking on it!

An obscure tip that might be useful
> You can hold Ctrl while selecting a beatmap to watch a perfect automated play through!

Another obscure tip
> Don't like a certain element in-game? Customise it using the \"Skin editor\"! Or by pressing Ctrl-Shift-S!

No need to say
> Are you sensitive to latency? Test your limits using the \"Latency Certifier\" in settings!

Basically unlimited will be used by hardcore stable players without any consideration
> Be sure to check out other \"Rulesets\" that the community has made!"

I'm unsure this should be added or not because rulesets aren't really that useful for the average user but still open things a bit

**Reply by peppy** [reply]
Added most

**Comment by nahkd123** [comment]
"You can drag and drop an image into Skin editor!"

**Reply by peppy** [reply]
Added!

**Comment by Stoppedpuma** [comment]
"You can change the visibility mode of the gameplay HUD by pressing Shift-Tab!"
"You can drag and drop a correctly named image into the skin editor to add it to your skin!"

**Comment by Skubers** [comment]
I think the tips should consist mainly of things that are new to Lazer, so here are some of my ideas:

"You can launch the First Run experience once again with the "Run setup wizard" button in the Settings"
"If you experience issues or have ideas for the game, be sure to visit official [github](https://github.com/ppy/osu) page of osu!"
"Your tablet can work now without any external drivers as osu! is natively supporting OpenTabletDriver in-game. Check Settings for more info!"
"osu!supporter gives you a possibility to see Storyboards of maps even in Main Menu! More benefits will be added in the future!"
"After playing a map and retrying it, osu! will automatically decide how much offset would be beneficial for your gameplay! Check out the Offset slider when you retry the map!"
"You can now change the scrolling direction for osu!mania to imitate the StepMania gameplay style!"
"There are a few graphical renderers to choose from if you're experiencing issues. If you don't know what they do, just stick with the Automatic one!"
"You can now decide if you want to show all information or to hide your identifiable information in Discord's Rich Presence!"
"You can import most things from osu!stable, from beatmaps and skins, to scores and collections!"
"If you accidentally deleted a beatmap, a difficulty or a mod preset, you can bring them back using a button in Maintenance section of the Settings!"
"You can check if you're actually seeing any difference in latency with our new Latency Certifier in Settings!"
"If your HUD is hidden during gameplay, you can have a peek by holding the CTRL key!"

**Reply by nagi-desuuu** [reply]
> "If you accidentally deleted a beatmap, a difficulty or a mod preset, you can bring them back using a button in Maintenance section of the Settings!"

https://github.com/ppy/osu/blob/58ea8f9920554c63b24c229b17d474db77412c5c/osu.Game/Screens/Menu/MenuTip.cs#L109C41-L109C41

**Reply by Skubers** [reply]
Yep, possible I missed this one when I wrote them

**Comment by LunaticalFrost** [comment]
"If you want to use the main manu more wise and gimmickly and doesn't hover over the buttons, press "P" to play and "M" if you want open the multiplayer, "E" to choose mode of edit (skin or map editor), "O" to open the settings menu quickly and "Q" to close the game with confirmation."

"You can always open the selection of modes with pressing F1 in Song Select menu!"

That all i have in mind to suggest for tips. For now.

**Comment by TommyGodlike** [comment]
"You can change your online appearing (or sign out) by clicking on your profile card at right-top!"
"Aside from "triangles", you can choose other two intro sequences: "welcome" from pre-2015 and "circles" from post-2015 era."
 
please check spelling

**Reply by nagi-desuuu** [reply]
```diff
-  appearing
+  appearance
```

**Comment by WhatDamon** [comment]
## stable's menu tips can be added in lazer directly
- Don't ignore unranked maps. If you find some you like and leave constructive comments, there is a much higher chance of them being ranked!
- Make sure to take breaks every so often. Your hands have to serve you in life for more than just osu! spinners.
- The wiki has a wealth of information in it. Seriously, read it.
- For fast combos, try toggling between left and right mouse button, or use keyboard alternative Z/X!
- You can play songs together with other players in multiplayer mode!
- Hit F8 at any time to activate the in-game chat window. You can check out the commands available in chat by typing !help or /help
- Take a screenshot from within osu! by hitting F12.

## a slightly modified version of stable's original
- Press F8 anywhere in game to join osu! chat, or press F9 to see the dashboard!
- Quickly retry a beatmap by holding Ctrl-R or " ` "(under the esc key) for a short time! You can also use a custom key.

## some menu tips for lazer only
- Press F6 to control media playback!
- osu! is being developed with the support of the open source community, and you can participate too!
- There are many community-developed "rulesets" in osu!, some of which provide new game modes, and some of which add new features to the game!
- Got a bug? Come to osu!'s GitHub repository to submit issues! [^1]
- Say goodbye to pressing F5 refreshing the beatmaps list!
- A lot of the components in osu! bounce along with the BPM of the music being played!
- If you find that some strings are not translated, go to osu!'s Crowdin and look for them!
- Experience lower chat latency in the new osu!
- Are you a mapper? Come experience the new beatmap editor, it's pretty cool!

-----

optionally, you can add some ;P

Of course I hope that the next step will be to allow the community to translate these sooner, aka add these strings to crowdin.

[^1]:One of the characters above that doesn't display properly is the github icon in font awesome and can be displayed in lazer

**Comment by Mux1c** [comment]
How about adding 1 joke prompt like some other games (League of Legends, CS:GO(previously), Minecraft) have, I'd suggest the running joke:
"You are here forever"

**Reply by N0Y0U2114** [reply]
another few:

1-2 1-2

727

When you see it

222.22

Ohio State University

**Reply by ziv-vy** [reply]
I think that's too many to be honest, and most people don't know what "222.22" or "1-2 1-2" means

**Reply by Aworldc** [reply]
"Clicking the circles gives more pp than not clicking the circles".

**Comment by ItzMeNeon** [comment]
## Osu Tips (May These Tips Can Be Compatible on osu! Stable Or Osu!Lazer)

"Having Issues With Your Tablet? Try Disabling The In-Game Tablet Usage And High Precision Mouse On The Settings > Input Menu"
"Play With The Others Can Give More Fun If You Play Multiplayer Mode!"
"Feel Bored After Playing Some Game In Osu? Try To Hop On The Chat Channel And Interact With The Others By Pressing F8 On Your Keyboard!"
"Always Use Your Main Account! We're Always Watching You From Afar :)"
"Feel Something Is Empty On Your Skins? Try Downloading Some Skins From Osuck's Website Or Any Other Website"
"Sometimes Osu!'s Main Server Could Be Slow To Download Beatmaps, So Try Using Other Download Mirrors Like Beatconnect etc"
"Always Be Nice People While Chatting Or Interacting With People, Because They Gonna Treat You The Same As You Did :D"
"Before Playing Osu, Try To Finish Your Work / Homework First... Seriously"
"Any Idea To Make Osu better? Try To Give Your Beautiful Answer On Osu's Github Discussion, Osu's Forum And Discord Server!"
"Don't Force Yourself To Play Osu! even Further... Take Some Break And Touch Grass For A While :D"
"Try To Share Your Gameplay With The Others By Saving Your Replay After A Game By Pressing F2!"

**I Hope Peppy Could Add These And Hopefully Didn't Upset Him/Her By My Ideas :D**

### Written by ShinoriTsurumi | AndrewID12 "[My Osu Profile](https://osu.ppy.sh/users/32753118)"

**Reply by ItzMeNeon** [reply]
If Some Tips Were Kinda Miss-spelled, try To Make The Tips More Understandable :D

**Reply by ziv-vy** [reply]
"Always use your main account! We're always watching you from afar ;)"

Peppy, pls.

**Comment by home-gihub** [comment]
"You can resize the chat window by draging the top"

**Comment by AImixAE** [comment]
# Osu! Tips (Chinese)

> 通过 ← → 键切换歌曲，也可以使用↑ ↓键切换难度

> 每日挑战具有极高难度哒！你确定不挑战一下？

> 戳泡泡可比造泡泡难多了！

> 通过 `Ctrl + B` 来选择你心仪的歌曲！

> 我们有着全新的 Tips 引擎

> Osu! Lazer 有着实时的PP显示！

> 打不过不要硬刚了！你还不知道 `mods` 吗？

**Comment by Shavixinio** [comment]
"You can quickly switch between game modes with Ctrl + 1-4"
"When randomizing a beatmap in the beatmap select menu, you can hold shift and a button for switching back to the previous song will appear"
It could be worded better, but I think most of you know what I mean.

**Comment by N0Y0U2114** [comment]
Want to practice only part of a map? Use the editor! Skip to the point you want to start at and and click test!

**Reply by peppy** [reply]
We're hoping to add a practice mode to avoid this being a thing (because the editor isn't really setup to practice specific mods, for instance).

**Reply by SirHonkkk** [reply]
looking forward to seeing practice mode. i could definitely use it lol

**Comment by ziv-vy** [comment]
Many open source projects include some kind of message informing users of the open-source status of said project and asking users to contribute to it. One wouldn't hurt for osu!

"osu! is open source! You can help make the game better at github.com/ppy/osu"

**Comment by SirHonkkk** [comment]
Ctrl+Shift+T to refresh tips. i like seeing tips when o start the game, and i think a way to refresh them would be nice.

**Reply by Joehuu** [reply]
Should probably be a button instead for mobile consideration. And Ctrl+Shift+T can potentially be something else (e.g. opening previous closed overlays / tabs like browsers).

**Reply by SirHonkkk** [reply]
good point. i didnt think of mobile users. also a list of all the tips would be cool

**Reply by Gabixel** [reply]
I guess an easy way to see all the strings is [from the code itself](https://github.com/ppy/osu/blob/master/osu.Game/Localisation/MenuTipStrings.cs) (see all the comments/summaries)

**Reply by SirHonkkk** [reply]
true

**Comment by SirHonkkk** [comment]
“Inspired by Elite Beat Agents!/Osu! Tatakae! Ouendan!”

**Comment by shiumano** [comment]
"What to see a perfect play of the beatmap that you are making? Press TAB during test play to enable Autoplay!"
thanks @Taoshix

**Reply by Taoshix** [reply]
"Want to switch to Autoplay while test playing your map? Press TAB to enable Autoplay!" is what I think this means

**Reply by shiumano** [reply]
I felt it would be better to do that it would be auto play.
I changed the wording a bit.

**Reply by Taoshix** [reply]
Could also say
"What to see a perfect play of the beatmap that you are making? Press TAB during test play to enable Autoplay!"

**Comment by Qunas101** [comment]
Dunno if it's an appropriate place for this since it's not really a "tip", but it would be nice to remind players to take breaks and not push themselves too hard since osu can very much lead to hand injuries. 

Something as simple as "Remember to take breaks!" is probably the best variant

Also, [this](https://github.com/ppy/osu/blob/master/osu.Game/Localisation/MenuTipStrings.cs) appears to be the new file. peppy plz fix

**Reply by Joehuu** [reply]
Your suggestion may potentially be in the beatmap loading screen, see https://github.com/ppy/osu/discussions/31090.

I have fixed the link.

**Comment by smit4k** [comment]
In regards to disabling mouse clicks during gameplay:

"Accidentally clicked your mouse while playing? You can disable clicks during gameplay in Settings, → Input"

**Comment by Realbulion** [comment]
After the last update (scoreboard score color to be specific) (this post [#35843](https://github.com/ppy/osu/pull/35843)) i would suggest making scoreboard score color changeable through a pool of colors in settings or in editor. For me i don't like the yellow color and would like to change it back to blue or purple if there would be smth like that. :)

**Comment by Akarinnnnn** [comment]
"Have you participated osu! (stable) tournaments? Lazer scoring is consist of 50% accuracy and 50% combo, instead of 30% accuracy and 70% combo in ScoreV2"

Chinese raw:
“你打过osu! (stable)的standard模式比赛吗？和ScoreV2不同，Lazer计分制由50%准确率分和50%连击分构成，而ScoreV2由30%准确率分和70%连击分构成”

Should find a place to mention this tip is talking about standard ruleset in English version(the “standard模式”).

**Reply by CalCobalt** [reply]
Lazer的记分模式好像就叫ScoreV3

**Comment by Geode-create** [comment]
"There's an issue about osu!? No problem! you can open an issue on github.com/ppy/osu/ or saying it on our Discord!"
